### PR TITLE
Website updated: in style.css /.social-media-icons/margin-left deleted & flex class added to index.html

### PR DESCRIPTION
**Pull request by priyankarpal**
**Website updated: in style.css /.social-media-icons/margin-left deleted & flex class added to index.html**

**All social media icons are now center**

Screenshot from 2023-01-07 21-35-24

### CODE REVIEW

File: dark/assets/css/styles.css [hunk 1]
```diff
@@ -752,7 +758,7 @@ html {
   margin: 50px;
   margin-bottom: 400px;
   display: flex;
-  margin-left: 460px;
+
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:761]: Can you delete this extra line @priyankarpal

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css (hunk 1 above)
Comment (priyankarpal) [reply dark/assets/css/styles.css:761]: ok

**Comment by priyankarpal** [comment]
**Check I updated.**

File: dark/assets/css/styles.css (hunk 1 above)
Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:762]: Can you delete this extra line @priyankarpal 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css [hunk 2]
```diff
@@ -1122,4 +1129,4 @@ html {
   border-radius: 2rem;
   font-size: 1.1rem;
   transition: 0.2 ease;
-}
+}
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:1132]: Add end of the line @priyankarpal 
```suggestion
}

```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html [hunk 3]
```diff
@@ -194,33 +213,36 @@ <h1>CONTACT ME</h1>
...
+          <div class="mb-3">
+            <textarea name="message" id="message" row="20" col="20" class="form-control bg-dark text-white"
+              placeholder="Enter Your Message"></textarea>
+          </div>
+          <button type="submit" class="btn btn-primary d-block m-auto">
+            Submit
+          </button>
+        </form>
       </div>
-    </section>
```
Comment (YoshithaRathnayake) [inline dark/index.html:223]: Why did you remove this section ending tag? @priyankarpal

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/index.html (hunk 3 above)
Comment (priyankarpal) [reply dark/index.html:223]: checking...

File: dark/assets/css/styles.css (hunk 1 above)
Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:762]:
````suggestion
````
Remove this line @priyankarpal

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
Now check my branch "Priyankarpal". I updated everything. You have to check it manually.

**Comment by priyankarpal** [comment]
I upgraded social icons positions, and upgraded buttons.

**Comment by priyankarpal** [comment]
# Don't forget to check my branch "priyankarpal"

File: dark/assets/css/styles.css [hunk 4]
```diff
@@ -29,7 +26,12 @@
   color: black;
   background: var(--primary-color);
 }
-
+/*  Utility */
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:29]: Add a new line before this @priyankarpal 
```suggestion

/*  Utility */
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css (hunk 2 above)
Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:1127]: Do these above mentioned changes @priyankarpal

File: dark/assets/css/styles.css (hunk 1 above)
Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:757]: Do these above mentioned changes @priyankarpal

File: dark/assets/css/styles.css [hunk 5]
```diff
@@ -751,8 +753,8 @@ html {
   top: 300px;
   margin: 50px;
   margin-bottom: 400px;
-  display: flex;
-  margin-left: 460px;
+
```
Comment (YoshithaRathnayake) [inline dark/assets/css/styles.css:756]: Remove this line too @priyankarpal 
```suggestion
```

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]

File: dark/assets/css/styles.css (hunk 4 above)
Comment (YoshithaRathnayake) [reply dark/assets/css/styles.css:29]: Don't resolve the conversation without doing the changes @priyankarpal 
Do these changes first

Comment (priyankarpal) [reply dark/assets/css/styles.css:29]: check I deleted my own branch. I change HTML and CSS code to main.
